marko.json

```diff
@@ -192,5 +192,27 @@
             "@selected": "boolean"
         }
     },
-    "<ebay-select-option>": {}
+    "<ebay-select-option>": {},
+    "<ebay-tab>": {
+        "renderer": "./src/components/ebay-tab/index.js",
+        "transformer": "./src/common/transformer/index.js",
+        "@*": "expression",
+        "@html-attributes": "expression",
```

Comment:
Do we really need `@html-attributes`? `@*` should be sufficient right? This is a general comment, not specific to this component.

Reply:
This was introduced to support dynamic pass-through attributes. More details here: https://github.com/eBay/ebayui-core/pull/154